grammar.y

```diff
@@ -1165,6 +1162,7 @@ numeric_literal:
   }
 | FLOAT
   {
+    yylex.(*Lexer).AddError(&ErrNumericLiteralFloat{Value: $1})
```

Comment:
We are still parsing literal float numbers but throwing an error now. Another option would be to not be able to understand float numbers and throw a syntax error. But that would imply a bigger and risker change in the lexer. I think this is fine. Maybe we will still need to parse floats for some reason, better to keep that logic.

Reply:
Agree.